src/collect/schemas/collect.schema.ts

```diff
@@ -0,0 +1,23 @@
+import { Prop, Schema, SchemaFactory } from "@nestjs/mongoose";
+import mongoose, { HydratedDocument } from "mongoose";
+
+export type CollectDocument = HydratedDocument<Collect>;
+
+@Schema({ timestamps: true, versionKey: false })
+export class Collect {
+  _id: mongoose.Schema.Types.ObjectId;
+
+  //User A
+  @Prop()
+  collect_user_email: string; //from guard
```

Comment:
What's the difference between collect_user_email and collected_user_email?

Reply:
collect_user (user A)  will collect/cancel collect collected_user's (user B) post.
and user email is the unique identified user id in our project.